### DISCUSSION TITLE
Replacement for jQuery UI Ajax Tabs

### DISCUSSION THREAD

**Discussion by subrayamallya** [twbs/bootstrap#31481] [General]
I have been trying to find implement a Bootstrap only version of the jQuery UI Ajax tabs (https://jqueryui.com/tabs/#ajax) where I could have each tab linked to a URL and loaded via Ajax and maintaining state of tab.

I know what Navs, Pills don't seem to do that or am I missing something?

**Comment by Deckluhm** [comment]
You can use Navs JS events to trigger AJAX query on tab change: https://getbootstrap.com/docs/4.5/components/navs#events
